src/app/components/media/MediaTypeDisplayIcon.js

```diff
@@ -52,32 +52,32 @@ export function mediaTypeFromFilename(fileName) {
   return mediaType;
 }
 
-export default function MediaTypeDisplayIcon({ mediaType }) {
+export default function MediaTypeDisplayIcon({ mediaType, fontSize = 'var(--iconSizeSmall)', color = 'var(--textPlaceholder)' }) {
   switch (mediaType) {
   case 'Claim':
-    return <Description style={{ fontSize: 'var(--iconSizeSmall)' }} />;
+    return <Description style={{ fontSize, color }} />;
   case 'Link':
-    return <Public style={{ fontSize: 'var(--iconSizeSmall)' }} />;
+    return <Public style={{ fontSize, color }} />;
   case 'UploadedImage':
-    return <Image style={{ fontSize: 'var(--iconSizeSmall)' }} />;
+    return <Image style={{ fontSize, color }} />;
   case 'UploadedVideo':
-    return <Movie style={{ fontSize: 'var(--iconSizeSmall)' }} />;
+    return <Movie style={{ fontSize, color }} />;
   case 'UploadedAudio':
-    return <Audiotrack style={{ fontSize: 'var(--iconSizeSmall)' }} />;
+    return <Audiotrack style={{ fontSize, color }} />;
   case 'Facebook':
-    return <Facebook style={{ fontSize: 'var(--iconSizeSmall)' }} />;
+    return <Facebook style={{ fontSize, color }} />;
   case 'Instagram':
-    return <Instagram style={{ fontSize: 'var(--iconSizeSmall)' }} />;
+    return <Instagram style={{ fontSize, color }} />;
   case 'Telegram':
-    return <Telegram style={{ fontSize: 'var(--iconSizeSmall)' }} />;
+    return <Telegram style={{ fontSize, color }} />;
   case 'Tiktok':
-    return <Tiktok style={{ fontSize: 'var(--iconSizeSmall)' }} />;
+    return <Tiktok style={{ fontSize, color }} />;
   case 'Twitter':
-    return <Twitter style={{ fontSize: 'var(--iconSizeSmall)' }} />;
+    return <Twitter style={{ fontSize, color }} />;
   case 'Youtube':
-    return <YouTube style={{ fontSize: 'var(--iconSizeSmall)' }} />;
+    return <YouTube style={{ fontSize, color }} />;
   case 'Blank':
-    return <PlaylistAddCheck style={{ fontSize: 'var(--iconSizeSmall)' }} />;
+    return <PlaylistAddCheck style={{ fontSize, color }} />;
```

Comment:
@danielevalverde please change the "Blank" icon from the one here to the new svg attached:

![empty_media](https://github.com/meedan/check-web/assets/418001/cd259f04-bcf3-4aa5-aaef-de16a389b8db)